tests/e2e/testPages/prebid-body-multi-players.html

```diff
@@ -101,7 +101,7 @@
 
 	    function pageNotification(type, data) {
 			if (type === 'event') {
-	    		appendStatistics('Event: ' + data);
+	    		appendStatistics('Event: ' + data, 1);
```

Comment:
can we add a comment stating the meaning of the "1" argument

Reply:
added comment